File: app/src/main/java/com/exercise/articlesloader/MainActivity.java
package com.exercise.articlesloader;

import androidx.appcompat.app.AppCompatActivity;

import android.content.Intent;
import android.os.Bundle;
import android.view.View;

public class MainActivity extends AppCompatActivity {

    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        setContentView(R.layout.activity_main);
    }

    public void openArticleFeed(View v){
        Intent intent = new Intent(MainActivity.this, ArticlesActivity.class);
        startActivity(intent);
    }
}

File: app/src/main/java/com/exercise/articlesloader/MainActivity.kt
package com.exercise.articlesloader

import android.content.Intent
import android.os.Bundle
import android.view.View
import androidx.appcompat.app.AppCompatActivity

class MainActivity : AppCompatActivity() {
    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        setContentView(R.layout.activity_main)
    }

    fun openArticleFeed(v: View?) {
        val intent = Intent(this@MainActivity, ArticlesActivity::class.java)
        startActivity(intent)
    }
}